utils/aeromatic++/Systems/Controls.cpp

```diff
@@ -143,6 +141,7 @@ void CableControls::set(const float* cg_loc)
     _aircraft->_CLalpha.at(1) = CLaw[1];
     _aircraft->_CLalpha.at(2) = CLaw[2];
 
+printf("CLalpha: %f\n", _aircraft->_CLalpha.at(0));
```

Comment:
Probably a debug message that should be removed ?

Reply:
Ah, yeas please. It's a debugging message indeed.